hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncAdminBuilder.java

```diff
@@ -36,34 +36,40 @@
    * Set timeout for a whole admin operation. Operation timeout and max attempt times(or max retry
    * times) are both limitations for retrying, we will stop retrying when we reach any of the
    * limitations.
-   * @param timeout
-   * @param unit
```

Comment:
Maybe better to add some javadoc to parameters here instead of removing? I mean after all we are changing this part, let's improve it.

Reply:
We do not have javadoc for these parameters in both `AsyncTableBuilder` and `TableBuilder`, and I think the parameter name is good enough to tell users the meaning?

Reply:
It will generate checkstyle error [by default](http://checkstyle.sourceforge.net/config_javadoc.html#JavadocMethod) although not a big deal.